### PR TITLE
Added db migration script to update ocrData column type to JSON

### PR DESCRIPTION
**Pull request by aliveni**
OcrData column in the envelope is not being used at the moment.
Another PR will be created to use this column and drop the existing column (`ocrDataJson`)

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/ScannableItem.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
     private Timestamp nextActionDate;
 
     @Type(type = "jsonb")
-    @Column(name = "ocrDataJson", columnDefinition = "jsonb")
+    @Column(name = "ocrData", columnDefinition = "jsonb")
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/ScannableItem.java]: We should only switch to the new column once the application has started storing OCR data in the right format

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/ScannableItem.java]: 👍

File: src/main/resources/db/migration/V025__Update_ocrData_column.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ALTER TABLE scannable_items
+  DROP COLUMN ocrData;
+
+ALTER TABLE scannable_items
+  ADD COLUMN ocrData JSON null
```
Comment (rkondratowicz) [inline src/main/resources/db/migration/V025__Update_ocrData_column.sql]: It took my a while to realise `ocrData` column is currently not used.
I think this migration deserves a comment, at least on the PR.

Comment (aliveni) [reply src/main/resources/db/migration/V025__Update_ocrData_column.sql]: I'll add a comment. 
We want to do this in 2 PRs. The first PR to add the field and the second PR will have the migration script to use to drop the existing `ocrDataJson` and use the field `ocrData`.

**Comment by lgonczar** [comment]
Why not merge this one?

File: src/main/resources/db/migration/V025__Update_ocrData_column.sql (hunk 2 above)
Comment (lgonczar) [reply src/main/resources/db/migration/V025__Update_ocrData_column.sql]: Dropping ocrDataJson alone will need to be done in a separate deployment, as the last change.

Comment (aliveni) [reply src/main/resources/db/migration/V025__Update_ocrData_column.sql]: 👍

**Pull request merged**